.gitignore

```diff
@@ -1,6 +1,7 @@
 /.idea/
 bin/
 cache/
+ignored/
```

Comment:
I wanted a general folder name where I could put stuff to be ignored.

I'm putting my downloaded `mondo.sssom.tsv` in here, since it comes from the `mondo` repo. Not sure if there's anywhere better to put this? I could also put it in a dir named `input/` and ignore that.

Reply:
Will do!